azure-pipelines/templates/sign.yml

```diff
@@ -0,0 +1,32 @@
+steps:
+  # Check if the SignExtension.signproj file exists and set a variable using PowerShell
+  # All other steps in this template will only run if the file exists
+  - powershell: |
+      $fileExists = Test-Path -Path "$(Build.SourcesDirectory)/SignExtension.signproj"
```

Comment:
Can we stuff this .signproj into `azure-pipelines` instead of at the root of the project? There's enough gunk at the root...

Reply:
I will try *cries in more pipelines trial and error*

Reply:
Got it first try 😎

Reply:
:shipit: